tests/wasm/mod.rs

```diff
@@ -190,102 +253,124 @@ fn no_export() {
 #[test]
 #[serial]
 fn echo() {
-    let mut input: Vec<u8> = Vec::with_capacity(2 * 1024 * 1024);
-
+    let mut input: Vec<_> = Vec::with_capacity(2 * 1024 * 1024);
     for i in 0..input.capacity() {
         input.push(i as _);
     }
     let input = input.as_slice();
 
-    let bin = env!("CARGO_BIN_FILE_ENARX_WASM_TESTS_echo");
-    run_wasm_test(bin, None, 0, input, input, None);
+    run_wasm_test(
+        env!("CARGO_BIN_FILE_ENARX_WASM_TESTS_echo"),
+        None,
+        0,
+        input,
+        input,
+        None,
+    );
```

Comment:
Back then I introduced `bin` to remove the line breaks

Reply:
Why, oh why did you add them back everywhere?

Reply:
`bin` is a redundant variable, which is only used once and duplicated for N test cases we have.
1. When reading the code, one has to check what `bin` is - look through the function and check for usages. That is not required when there is no variable.
2. It's faster to duplicate and adapt a single invocation, rather than a variable binding and an invocation (note that it has to be done for N test cases)

I disagree, but I will add a `const` here